packages/engine/src/hedronTypes.ts

```diff
@@ -0,0 +1,23 @@
+import * as THREE from 'three'
+import * as THREE_WEBGPU from 'three/webgpu'
+import * as THREE_TSL from 'three/tsl'
+import * as POSTPROCESSING from 'postprocessing'
+
+/**
+ * Type definitions for global Hedron namespace
+ */
+export interface HedronGlobal {
+  dependencies: {
+    THREE: typeof THREE
+    THREE_WEBGPU: typeof THREE_WEBGPU
+    THREE_TSL: typeof THREE_TSL
+    POSTPROCESSING: typeof POSTPROCESSING
+  }
+  fs?: any // fs is only available when running on desktop
+}
+
+declare global {
+  interface Window {
+    __HEDRON: HedronGlobal
+  }
+}
```

Comment:
I'm not so keen on promoting the hacky `__HEDRON` namespace. The idea behind it is just as a middle step to mock package imports in sketches (e.g. `import { MeshNormalMaterial } from 'three'`). The moment we start typing this and tacking on extra properties I feel like we're heading down a messy path...